src/sea-lib-mapping.ts

```diff
@@ -253,6 +280,63 @@ export function handleAuctionBid(event: AuctionBid): void {
     projectMinterConfig,
     event.block.timestamp
   );
+
+  // Update Bids entity
+  const bidId =
+    event.address.toHexString() +
+    "-" +
+    event.params.bidder.toHexString() +
+    "-" +
+    event.block.timestamp.toString() +
+    "-" +
+    event.params.tokenId.toString();
+
+  let bid = Bid.load(bidId);
+  if (bid) {
+    // This is never expected to be possible with SEA minter
+    log.warning("Existing Bid with ID {} found for SEA Minter contract {}", [
+      bidId,
+      event.address.toString()
+    ]);
+  } else {
+    // New bid
+    bid = new Bid(bidId);
+    // Create new account entity if one for the bidder doesn't exist
+    const bidderAccount = new Account(event.params.bidder.toHexString());
```

Comment:
do we need to first try and load the account?

Reply:
I think that if the account already exists it just overwrites the existing id with the same value, saving us from having to call load.. LMK if I'm missing anything here @ryley-o

Reply:
yup! I think this is how we commonly deal with accounts since it only has an ID and derived fields 👍